loom/script/native/lsNativeInterface.cpp

```diff
@@ -204,7 +204,8 @@ void NativeInterface::resolveScriptType(Type *type)
         }
     }
 
-    lmAssert(0, "Unable to resolve script type %s", type->getFullName().c_str());
+    type->setMissing("resolveScriptType");
```

Comment:
Yeah but why pass the string if it's not used? Should it be an enum if one of only a few fixed values?